recipe/meta.yaml

```diff
@@ -12,12 +12,13 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
-  skip: true  # [win or not py2k]
+  number: 2
+  skip: True  # [win and py3k or win64 or not py2k]
```

Comment:
Don't worry. We have a lot of moving parts. Feedback on how to improve is welcomed :wink: